### PR TITLE
FluentbitDropRatio alert

### PR DESCRIPTION
**Pull request by hervenicol**
This PR:

- add FluentbitDropRatio alert
Towards https://github.com/giantswarm/giantswarm/issues/22920

### Checklist

- [x] Update changelog in CHANGELOG.md.
- [ ] Request review from oncall area, as well as team (e.g: `oncall-kaas-cloud` GitHub group).
- [x] Alerting rules must have a comment documenting why it needs to exist.
- [ ] Consider creating a dashboard (if it does not exist already) to help oncallers monitor the status of the issue.

### CODE REVIEW

File: helm/prometheus-rules/templates/alerting-rules/fluentbit.rules.yml [hunk 1]
```diff
@@ -28,6 +29,24 @@ spec:
         severity: notify
         team: atlas
         topic: observability
+    - alert: FluentbitDropRatio
+      annotations:
+        description: '{{`Fluentbit ({{ $labels.instance }}) is dropping more than 1% records.`}}'
+        opsrecipe: fluentbit-too-many-erros/
+      # Check the ratio of dropped records over the total number of records.
+      expr: rate(fluentbit_output_dropped_records_total[10m]) / (rate(fluentbit_output_proc_records_total[10m]) + rate(fluentbit_output_dropped_records_total[10m])) > 0.01
+      for: 10m
```
Comment (mcbenjemaa) [inline helm/prometheus-rules/templates/alerting-rules/fluentbit.rules.yml:38]: Sure for only 10m

Comment (hervenicol) [reply helm/prometheus-rules/templates/alerting-rules/fluentbit.rules.yml:38]: Same as the previous `FluentbitTooManyErrors` alert.
This way, they should both pop at the same time, and correlation is easier.

Comment (hervenicol) [reply helm/prometheus-rules/templates/alerting-rules/fluentbit.rules.yml:38]: Same as the previous `FluentbitTooManyErrors` alert.
This way, they should both pop at the same time, and correlation is easier.

**Review by mcbenjemaa — APPROVED** [review]

**Comment by mcbenjemaa** [comment]
Please update the branch

**Pull request merged**